docs/WindowsBuild.md

```diff
@@ -78,15 +78,16 @@ git clone https://github.com/compnerd/swift-build swift-build
 
 ## Acquire ICU, SQLite3, curl, libxml2 and zlib
 
-Go to [compnerd's swift-build azure page](https://dev.azure.com/compnerd/swift-build/_build) and open [Pipelines](https://dev.azure.com/compnerd/swift-build/_build) where you'll see bots (hopefully green) for:
-
-- [ICU](https://dev.azure.com/compnerd/swift-build/_build?definitionId=9)
-- [SQLite](https://dev.azure.com/compnerd/swift-build/_build?definitionId=12&_a=summary)
-- [curl](https://dev.azure.com/compnerd/swift-build/_build?definitionId=11&_a=summary)
-- [libxml2](https://dev.azure.com/compnerd/swift-build/_build?definitionId=10&_a=summary)
-- [zlib](https://dev.azure.com/compnerd/swift-build/_build?definitionId=16&_a=summary)
+```
+python -m pip install --user msrest azure-devops tabulate
```

Comment:
FWIW `python` does nothing for me, I have to do `C:\Python27\python.exe

Reply:
Ugh, okay, lets change that to `C:\Python27\python.exe` then; this is part of the problem - python needs to be by absolute path :-(.